homework-objects.js

```diff
@@ -0,0 +1,70 @@
+// 1 Задача про обчислення різниці часу
+
+function periodOfTime(
+  startDate = '01 Jan 1990',
+  endDate = '15 Mar 2023',
+  timeCounter = 'hours'
+) {
+  const timePeriod = {
+    days: 86400,
+    hours: 3600,
+    minutes: 60,
+    seconds: 1,
+  };
+
+  const starting = Date.parse(startDate);
+  const ending = Date.parse(endDate);
+  const diffInSeconds = Math.abs(starting - ending) / 1000;
+  const difference = diffInSeconds / timePeriod[timeCounter];
+  return difference;
+}
+
+console.log(periodOfTime('02 Aug 1985', '03 Aug 1985', 'minutes'));
+console.log(periodOfTime('02 Aug 1985', '03 Aug 1985', 'hours'));
+console.log(periodOfTime('02 Aug 1985', '03 Aug 1985', 'seconds'));
+console.log(periodOfTime('31 Jan 2022', '03 Feb 2021', 'days'));
+console.log(periodOfTime());
+
+// 2
+
+const priceData = {
+  Apples: '23.4',
+  BANANAS: '48',
+  oRAngGEs: '48.7584',
+};
+
+function optimizer(data) {
+  const rrr = Object.entries(data)
```

Comment:
`rrr` is a strange name for the variable. You can even remove it (no need to use any variable at all)